packages/fxa-payments-server/.storybook/components/MockApp.tsx

```diff
@@ -105,4 +106,17 @@ export const MockApp = ({
   );
 };
 
+type MockPageProps = {
+  locale: string;
+  children: React.ReactNode;
+};
+
+export const MockPage = ({ locale, children }: MockPageProps) => {
```

Comment:
What is the purpose of `MockPage` here? It seems the `Localized` wrapper doesn't work properly without it for the `PaymentConsentCheckbox`, though it looks the same when I wrap with the existing `MockApp` instead?

Edit: I see this was moved from the `PaymentForm` component. Do you know why it was needed there?

Reply:
I'm not sure I understand the question.

`MockPage` is just a wrapper around `MockApp` that provides the locale in order for the `<Localized>` component to work inside storybook.

Are you saying that you don't like the way the code is broken up here?

Reply:
I'm saying I tried this out with wrapping `MockApp` instead of `MockPage` and it seemed to work, so why do we need `MockPage` here specifically?

```diff
diff --git a/packages/fxa-payments-server/src/components/PaymentConsentCheckbox/index.stories.tsx b/packages/fxa-payments-server/src/components/PaymentConsentCheckbox/index.stories.tsx
index 3b6f906ef..90a8e2075 100644
--- a/packages/fxa-payments-server/src/components/PaymentConsentCheckbox/index.stories.tsx
+++ b/packages/fxa-payments-server/src/components/PaymentConsentCheckbox/index.stories.tsx
@@ -3,7 +3,7 @@ import { storiesOf } from '@storybook/react';
 import { PaymentConsentCheckbox } from './index';
 import { Form } from '../fields';
 import useValidatorState from '../../lib/validator';
-import { MockPage } from '../../../.storybook/components/MockApp';
+import { MockApp } from '../../../.storybook/components/MockApp';
 import { SELECTED_PLAN } from '../../lib/mock-data';
 
 const WrapCheckbox = () => {
@@ -17,10 +17,10 @@ const WrapCheckbox = () => {
 
 storiesOf('components/PaymentConsentCheckbox', module).add('default', () => (
   <div style={{ display: 'flex' }}>
-    <MockPage locale="auto">
+    <MockApp>
       <div className="product-payment">
         <WrapCheckbox />
       </div>
-    </MockPage>
+    </MockApp>
   </div>
 ));
```

Then I was asking a broader question about why `MockPage` is needed at all anywhere -- why is `MockApp` insufficient?

Reply:
It looks like `MockApp` is sufficient if it worked for you, I can remove it from both payment form and paymentconsentcheckbox